lib/internal/util/inspect.js

```diff
@@ -613,16 +630,26 @@ wellKnownPrototypes.set(ArrayPrototype, { name: 'Array', constructor: Array });
 wellKnownPrototypes.set(ArrayBufferPrototype, { name: 'ArrayBuffer', constructor: ArrayBuffer });
```

Comment:
Should these include `__proto__: null`?

Reply:
We only look up these specific two properties, so I would not know how that could possibly make any difference.

Reply:
it should in theory, to avoid a future footgun, but I agree it won't matter in practice.

Reply:
I would keep it as is for now, if that's fine for you. I don't think we benefit from the change here.